evalml/tests/automl_tests/test_automl.py

```diff
@@ -5214,3 +5215,44 @@ def test_automl_chooses_engine(engine_choice, X_y_binary):
             automl = AutoMLSearch(
                 X_train=X, y_train=y, problem_type="binary", engine=engine_choice
             )
+
+
+def test_graph_automl(X_y_multi):
```

Comment:
I'm not sure we need to test this on the automl level as a unit test. It seems like if we can test it on the pipeline level, that should be sufficient coverage?

Reply:
The purpose of this test is to make sure that the `automl_parameters` that were set in `AutoMLSearch` are being picked up properly by `graph_json` in `{"Nodes": {"Attributes": ...}}`